python/mxnet/contrib/autograd.py

```diff
@@ -19,19 +20,27 @@ def set_recording(recording):
     check_call(_LIB.MXAutogradSetRecording(
         ctypes.c_int(recording)))
 
-def mark_variables(variables):
+def mark_variables(variables, grad_reqs, gradients):
```

Comment:
how about change to `mark_variables(variables, gradients, grad_reqs=None)`, then `if grad_reqs is None: grad_reqs = ['write']*size`